packages/react-vapor/src/components/iconCard/IconCard.tsx

```diff
@@ -0,0 +1,117 @@
+import classNames from 'classnames';
+import * as React from 'react';
+import {slugify} from 'underscore.string';
+
+import {SlideY} from '../../animations';
+import {TooltipPlacement} from '../../utils';
+import {Badge, IBadgeProps} from '../badge/Badge';
+import {Svg} from '../svg/Svg';
+import {ITooltipProps, Tooltip} from '../tooltip';
+
+export interface IconCardChoice {
+    value: string;
+    label: string;
+    icon?: string;
+    disabled?: boolean;
+}
+
+export interface IconCardProps {
+    title: string;
+    svgName: string;
+    badges?: IBadgeProps[];
+    description?: string;
+    onClick?: (choice?: string) => void;
+    tooltip?: ITooltipProps;
+    choices?: IconCardChoice[];
+    small?: boolean;
+}
+
+export const IconCard: React.FunctionComponent<IconCardProps & Omit<React.HTMLProps<HTMLDivElement>, 'onClick'>> = ({

```

Comment:
why do you pass the 'onClick' here if it's also at line 34? :eyes:

Reply:
Here I don't "pass" it, I `Omit` it from the type `React.HTMLProps<HTMLDivElement>`, because otherwise `onClick` is typed as `(choice?: string) => void | React.MouseEventHandler<HTMLButtonElement>`